### PR TITLE
Update invoke-pester help to improve #1634

### PR DESCRIPTION
**Pull request by asears**
Signed-off-by: asears <asears@users.noreply.github.com>

## 1. General summary of the pull request

A start to review / update for improvements to Invoke-Pester help.
Improves #1634

Requires some additional parameter definitions, examples, detailed information on Pester v5 Configuration object.

### CODE REVIEW

File: src/Pester.ps1 [hunk 1]
```diff
@@ -253,6 +253,189 @@ function Has-Flag {
...
+    -- EndLine (E): Performs code coverage analysis ending with the specified line.
+    Default is the last line of the script.
+
+    .PARAMETER CodeCoverageOutputFile
+    The path where Invoke-Pester will save formatted code coverage results file.
+    The path must include the location and name of the folder and file name with
+    a required extension (usually the xml).
+    If this path is not provided, no file will be generated.
+
+    .PARAMETER CodeCoverageOutputFileEncoding
```
Comment (asears) [inline src/Pester.ps1:341]: Update CodeCoverageOutputFileEncoding

File: src/Pester.ps1 [hunk 2]
```diff
@@ -253,6 +253,189 @@ function Has-Flag {
...
+    .PARAMETER CodeCoverageOutputFileFormat
+    The name of a code coverage report file format.
+    Default value is: JaCoCo.
+    Currently supported formats are:
+    - JaCoCo - this XML file format is compatible with Azure Devops, VSTS/TFS
+
+    The ReportGenerator tool can be used to consolidate multiple reports and provide code coverage reporting.
+    https://github.com/danielpalme/ReportGenerator
+
+    .PARAMETER Configuration
```
Comment (asears) [inline src/Pester.ps1:352]: Update Configuration.  Add separate help for Pester 5 Configuration Object

File: src/Pester.ps1 [hunk 3]
```diff
@@ -253,6 +253,189 @@ function Has-Flag {
...
+
+    .PARAMETER EnableExit
+    Will cause Invoke-Pester to exit with a exit code equal to the number of failed
+    tests once all tests have been run. Use this to "fail" a build when any tests fail.
+
+    .PARAMETER ExcludePath
```
Comment (asears) [inline src/Pester.ps1:358]: Update ExcludePath

File: src/Pester.ps1 [hunk 4]
```diff
@@ -253,6 +253,189 @@ function Has-Flag {
...
+
+    .PARAMETER ExcludeTagFilter
```
Comment (asears) [inline src/Pester.ps1:360]: Update ExcludeTagFilter

File: src/Pester.ps1 [hunk 5]
```diff
@@ -253,6 +253,189 @@ function Has-Flag {
...
+
+    .PARAMETER FullNameFilter
```
Comment (asears) [inline src/Pester.ps1:362]: Update FullNameFilter

File: src/Pester.ps1 [hunk 6]
```diff
@@ -253,6 +253,189 @@ function Has-Flag {
...
+
+    .PARAMETER Output
```
Comment (asears) [inline src/Pester.ps1:364]: Update Output

File: src/Pester.ps1 [hunk 7]
```diff
@@ -253,6 +253,189 @@ function Has-Flag {
...
+    Fails - to write everything except Passed (but including Describes etc.).
+    A common setting is also Failed, Summary, to write only failed tests and test summary.
+    This parameter does not affect the PassThru custom object or the XML output that
+    is written when you use the Output parameters.
+
+    .PARAMETER Strict
+    Makes Pending and Skipped tests to Failed tests. Useful for continuous
+    integration where you need to make sure all tests passed.
+
+    .PARAMETER TagFilter
```
Comment (asears) [inline src/Pester.ps1:419]: Update TagFilter

File: src/Pester.ps1 [hunk 8]
```diff
@@ -253,6 +253,189 @@ function Has-Flag {
...
+
+    .EXAMPLE
```
Comment (asears) [inline src/Pester.ps1:421]: Add Pester 5 Examples.  A separate Invoke-PesterTests.ps1 script containing configuration settings would be helpful

File: src/Pester.ps1 [hunk 9]
```diff
@@ -253,6 +253,189 @@ function Has-Flag {
...
+    This command runs all *.Tests.ps1 files in the current directory and its subdirectories.
+
+    .EXAMPLE
+    Invoke-Pester -Path .\Util*
+
+    This commands runs all *.Tests.ps1 files in subdirectories with names that begin
+    with 'Util' and their subdirectories.
+
+    .LINK
+    https://pester.dev/docs/quick-start
```
Comment (asears) [inline src/Pester.ps1:433]: Add docs link for Invoke-Pester

**Comment by nohwnd** [comment]
Thank you!

**Pull request merged**